README.md

```diff
@@ -371,6 +371,50 @@ The last step is the easiest, just hit the following command in your terminal, b
 npm start db.seed
 ```
 
+## Run in Docker container
```

Comment:
Add a section about how to configure the app settings. If a CI is deploying the project there isn't any `.env` file so we have to pass the env vars.

Reply:
Done